packages/shared-ux/button_toolbar/src/buttons/icon_button_group/icon_button_group.test.tsx

```diff
@@ -22,6 +22,12 @@ describe('<IconButtonGroup />', () => {
             onClick: jest.fn(),
             iconType: 'visText',
           },
+          {
```

Comment:
Why do we need this addition to the tests?

Reply:
Oh! I think I added that when I was debating adding another test to this suite, but then didn't end up doing that - good catch. I'll remove this :+1: